xwiki-commons-core/xwiki-commons-websocket/pom.xml

```diff
@@ -38,33 +38,38 @@
   </properties>
   <dependencies>
     <dependency>
-      <groupId>${project.groupId}</groupId>
+      <groupId>org.xwiki.commons</groupId>
```

Comment:
How is this change related to the move to jakarta?

Reply:
I doubt it is needed. I guess I just modified those by reflex for consistency.